crawl-ref/source/dat/des/branches/lair.des

```diff
@@ -2748,6 +2778,107 @@ MAP
   -------------------
 ENDMAP
 
+NAME:   pdpol_lair_dawg_pound
+TAGS:   no_monster_gen
+DEPTH:  Lair
+KMONS:  1 = wolf / warg w:5
+KMONS:  2 = hell hound / raiju
+KITEM:  2 = *
+SUBST:  _ = x _:3, S = S:1 . %:2
+NSUBST: s = 1:S / *= . S:1, D = 2:2 / 4= 1 2:1 / *= . 1:2
+: dgn.delayed_decay(_G, 'S', "felid skeleton")
+CLEAR:  _
+MAP
+            ________
+          __xxxxxxx__
+          _xx.SS..x__
+      _____x..SS..x__
+    ___xxxxx....DDx__
+ ____xxx..SS..DDxxx__
+ _xxxx.....SxDxxxxx__
+_xxSS....x..xxxx____
+_xDSS..Dxxssxxxx__
+_xS..Dxxxssxx____
+_xxDxxxxssxxx_
+__xxxxx..xx_
+    xx@@xx
+ENDMAP
+
+# Dead by dawn!
+NAME:    pdpol_lair_cabin_in_the_woods
```

Comment:
Really like what this vault is going for. However, it's got a clear runed door that does not reveal the part of the vault that's actually dangerous - this is a pretty bad spoiler, especially given the monsters you're placing. You could fix this either by putting the runed door on the inside room that gets the scary monster (using some sufficiently tricky substs or shuffles) or by putting windows into every room on the outside (and adding enough space to walk all the way around it).

Reply:
Yeah totally fair, I thought it might be permissible because of a small handful of other spoilery ones (I think there's one where you can't quite see a boulder beetle in the back that I like) but it's probably pretty mean with these monsters and the size of this thing.

I'll try out both those suggestions, I wouldn't have thought to do the runed door on just the monster's room but that could be a fun way to create the horror sense I was going for by walking around a creepy cabin before you see the thing. But if I'm being real people are probably going to otab right through it anyway lol.

Reply:
I think I got the runed doors just around the monster room working pretty nicely - it's kind of a mindbender (at least for me) but I think the result is good. One of the hard things about it was that originally I tried to do a line like `SHUFFLE: Maed/?ghf/Hgfh` to get all three rooms to swap in one line but I realized that can't work because we might end up with no `a` glyph and then an isolated room.